settings.php

```diff
@@ -71,8 +71,9 @@
  * updates" permission or the site maintenance account (the account that was
  * created during installation), you will need to modify the access check
  * statement below. Change the FALSE to a TRUE to disable the access check.
- * After finishing the upgrade, be sure to open this file again and change the
- * TRUE back to a FALSE!
+ *
+ * IMPORTANT!: After finishing the upgrade, be sure to edit this file again and
+ * change the TRUE back to a FALSE.
```

Comment:
Making this sentence more evident is a good change. 👍
I would not use an all-capitalized word, and I would avoid an exclamation point followed by a colon. Either the colon is used (this is my preference) or the exclamation point is used.

I would also avoid to use articles before `TRUE` and `FALSE`. (It is not clear why it should be *the TRUE* and *a FALSE*.)